components/FooterSlackCTA/FooterSlackCTA.tsx

```diff
@@ -1,19 +1,18 @@
-import Image from 'next/image';
 import styles from './FooterSlackCTA.module.scss';
 
 export const FooterSlackCTA = () => (
     <div className={styles.container}>
         <div className={styles.emoji}>👂</div>
-        <a
-            className={styles.link}
-            href="https://github.com/wagtail/wagtail/wiki/Slack"
-        >
+        <div>
             <div className={styles.cta__text}>Hear the latest news…</div>
             <div className={styles.cta}>
-                Join <strong className={styles.cta__tag}>#headless</strong> on
-                Slack
+                Join{' '}
+                <a href="https://github.com/wagtail/wagtail/wiki/Slack">
+                    <strong className={styles.cta__tag}>#headless</strong>
+                </a>{' '}
```

Comment:
Apologies for suggesting the refactor of this link element previously, looks like this was decided against. This might not obviously be a link to users as it has no underscore, although the hashtag and bold font weight should probably indicate it's a link sufficiently.